.github/workflows/api-publish.yaml

```diff
@@ -1,14 +1,18 @@
-# Dummy github action to allow iteration.
+# This workflow builds a docker image with jib and pushes it to Github Container Repository.
```

Comment:
GCR == Google Container Registry, i.e. gcr.io. I was surprised when you mentioned pushing containers to GitHub, but perhaps that was a mixup.

Apparently there is also a GitHub container registry @ `ghcr.io`, though I've never actually seen an image from here in the wild.

Reply:
Oops, stale comment. I looked at the github container registry, but DataBiosphere doesn't currently allow anyone to publish public packages there, so went with GCR.